File: KoKoslan/src/kokoslan/java/ast/KoKoId.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.java.ast;
import kokoslan.java.eval.*;


public class KoKoId extends KoKoAtom<String>{
   public KoKoId(String value){
      super(value);
   }
   @Override 
   public KoKoValue eval(KoKoContext ctx){
	   return ctx.find(this);
   }
   
}

File: KoKoslan/src/kokoslan/kotlin/ast/KoKoId.kt
/** 
   Andrey Arguedas Espinoza
   Daniela Armas Sanchez
   Michael Chen Wang
   Kimberly Olivas Delgado
*/

package kokoslan.kotlin.ast

import kokoslan.kotlin.eval.*

class KoKoId(value : String) : KoKoAtom<String>(value) {

    override fun eval(ctx : KoKoContext) : KoKoValue {
        return ctx.find(this)
    }
   
}